packages/core/src/makeStyles.ts

```diff
@@ -25,10 +27,22 @@ export function makeStyles<Slots extends string | number>(stylesBySlots: StylesB
     if (isLTR) {
       if (ltrClassNamesForSlots === null) {
         ltrClassNamesForSlots = reduceToClassNameForSlots(classesMapBySlot, dir);
+
+        process.env.NODE_ENV !== 'production' &&
+          isDevToolsEnabled &&
+          Object.entries(ltrClassNamesForSlots!).forEach(([slotName, sequenceHash]) => {
+            MK_DEBUG.addSequenceDetails(sequenceHash as SequenceHash, slotName);
+          });
```

Comment:
```suggestion
   MK_DEBUG.addSequenceDetails(ltrClassNamesForSlots);
```

Is there sense to change signature of `addSequenceDetails` to handle this instead? (few lines below the loop happens for `rtlClassNamesForSlots`)